django_project/core/settings/dev_rischan.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
```

Comment:
I think we can actually get rid of a lot of these dev_**user** profiles and replace with one ``dev_docker`` profile - what do you think?